lib/generators/shopify_app/install/templates/shopify_app.rb

```diff
@@ -9,6 +9,7 @@
   config.after_authenticate_job = false
   config.api_version = "<%= @api_version %>"
   config.shop_session_repository = 'ShopifyApp::InMemoryShopSessionStore'
+  config.allow_jwt_authentication = false
```

Comment:
I'd probably keep it out of the template for now since we don't want people to be using it yet. We can add it in to the template when we're ready for third-party rollout.